### PR TITLE
chore(repo): update maintenance status

### PR DESCRIPTION
**Pull request by rwaskiewicz**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build (`npm run build`) was run locally for affected output targets
- [ ] Tests (`npm test`) were run locally and passed
- [ ] Prettier (`npm run prettier`) was run locally and passed

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue URL:

## What is the new behavior?

this commit updates the maintenance status of the project across the README, issue template, and pull request template

feel free to push back on the verbiage!

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by christian-bromann — APPROVED** [review]
LGTM 👍

**Review by amandaejohnston — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ## Pull request checklist
 
+<!-- Please note that this repository is largely maintained for the purposes of supporting the Ionic Framework -->
+<!-- As a result, we may not immediately get around to reviewing pull requests that do not support the Framework (or at all) -->
```
Comment (tanner-reits) [inline .github/PULL_REQUEST_TEMPLATE.md:4]:
```suggestion
<!-- As a result, we may not immediately (or ever) get around to reviewing pull requests that do not support the Framework -->
```

**Review by tanner-reits — APPROVED** [review]
One possible small re-word

**Review by liamdebeasi — APPROVED** [review]

**Review by sean-perkins — APPROVED** [review]
Clear is kind, love it!

**Pull request merged**